app.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "openscope",
+  "name": "openScope",
```

Comment:
this should stay lowercase as I believe this maps directly back to our app name on heroku